core/types/transaction.go

```diff
@@ -55,6 +77,8 @@ type txdata struct {
 	Amount       *big.Int        `json:"value"      gencodec:"required"`
 	Payload      []byte          `json:"input"      gencodec:"required"`
 
+	TxType TransactionType `json:"transactionType" gencodec:"required"`
```

Comment:
Why do we need to add this extra field? The same information is derived from "ShardID" and "ToShardID". I think this field is redundant inside the transaction data.

Reply:
Yes, I am refactoring the code and removing the txType.